CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Features
 - Accessibility for menu divider @jurokapsiar ([#822](https://github.com/stardust-ui/react/pull/822))
+- Add `actionMenu` prop to `ChatMessage` component @layershifter ([#811](https://github.com/stardust-ui/react/pull/811))
```

Comment:
just putting my vote on that: would rather prefer `actions` than `actionsMenu` for the following reasons
- it will be easier to avoid breaking changes in future - for example, if at some point it will become an `actionsPanel`
- it is consistent with the naming taken for `Attachment` component, where there is an `action` slot

Reply:
but given the example from description, agree that it is definitely better to use singular `menu` noun in the name of the prop, to properly reflect underlying semantics

Reply:
I can be wrong but this decision comes from consistency point: you can pass only `ShorthandValue` or `ShorthandValue[]`. However, I understand your point, had the same feeling about it.

Reply:
got it, thank you @layershifter! actually, this original problem (`actions` and necessity to compromise on `actionsMenu`) is a good example that might serve as an inspiration for us to further refine this moment in future